pennylane/ops/qubit.py

```diff
@@ -1681,6 +1680,61 @@ def adjoint(self):
         new_phi = (np.pi - lam) % (2 * np.pi)
         return U3(theta, new_phi, new_lam, wires=self.wires)
 
+class IsingXX(Operation):
+    r"""IsingXX(phi, wires)
+    Ising XX coupling gate
+
+    .. math:: \mathtt{XX}(\phi) = \begin{bmatrix}
+            \cos(\phi / 2) & 0 & 0 & -i \sin(\phi / 2) \\
+            0 & \cos(\phi / 2) & -i \sin(\phi / 2) & 0 \\
+            0 & -i \sin(\phi / 2) & \cos(\phi / 2) & 0 \\
+            -i \sin(\phi / 2) & 0 & 0 & \cos(\phi / 2)
+        \end{bmatrix}.
+
+    **Details:**
+
+    * Number of wires: 2
+    * Number of parameters: 1
+    * Gradient recipe: :math:`\frac{d}{d \phi} \mathtt{XX}(\phi) = \frac{1}{2} \left[ \mathtt{XX}(\phi + \pi / 2) + \mathtt{XX}(\phi - \pi / 2) \right]`
+
+    Args:
+        phi (float): the phase angle
+        wires (int): the subsystem the gate acts on
+    """
+    num_params = 1
+    num_wires = 2
+    par_domain = "R"
+    grad_method = "A"
+
+    @classmethod
+    def _matrix(cls, *params):
+        phi = params
```

Comment:
It might be that `params` needs to be unpacked (it's a `tuple` atm). That might be the reason for the error in the test:
https://github.com/PennyLaneAI/pennylane/pull/1194/checks?check_run_id=2556530038#step:10:224
```suggestion
        phi = params[0]
```